fortnox-api.gemspec

```diff
@@ -16,6 +16,9 @@ Gem::Specification.new do |spec|
                        'easily persistable to Fortnox again using the `save` method.'
   spec.homepage      = 'http://github.com/accodeing/fortnox-api'
   spec.license       = 'LGPL-3.0'
+  spec.metadata = {
+    'rubygems_mfa_required' => 'true'
```

Comment:
Doesn't like the newer hash syntax? `rubygems_mfa_required: 'true'`, also should this be a string `'true'` or the singleton `true`?

Reply:
Keys must be strings since they are parsed to json. Same with the value, must also be a string: `metadata['rubygems_mfa_required'] value must be a String (Gem::InvalidSpecificationException)`